### ISSUE TITLE
Overshooting occurs in MNIST with Adam Optimizer with learning rate 0.01

### ISSUE THREAD

**Issue by taeguk** [taeguk/tensorflow-study#1]
![image](https://cloud.githubusercontent.com/assets/9107889/17075746/947894e8-50d8-11e6-994d-73a9bad7fba0.png)
source -> https://github.com/taeguk/tensorflow-study/commit/2a4793200fba1025a3e8221968f6463b3cff4d61
